src/scripts/Makefile

```diff
@@ -0,0 +1,47 @@
+# For more details about the translation jobs, see https://github.com/edx/frontend-i18n/blob/master/docs/how_tos/i18n.rst
+
+langs = "ar,fr,es_419,zh_CN"
+transifex_utils = ./node_modules/.bin/transifex-utils.js
+transifex_input = ./src/i18n/transifex_input.json
+tx_url1 = https://www.transifex.com/api/2/project/edx-platform/resource/$(transifex_resource)/translation/en/strings/
+tx_url2 = https://www.transifex.com/api/2/project/edx-platform/resource/$(transifex_resource)/source/
+
+# this directory must match .babelrc
+temp = ./babel-plugin-react-intl-temp
```

Comment:
Hmm, I wonder if this should be per-repo, so there isn't this cross-repo dependency?  In practice I don't see why anyone would every change this, though.

Reply:
Agreed, I don't really see why anyone would change it... I do wonder if it should be something like `./temp/babel-plugin-react-intl`?  That way people would just see a "temp" directory in their tree if they ran it.